ui/App/DiagnosticsScreen/Json.svelte

```diff
@@ -0,0 +1,44 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="ts">
+  export let data: unknown;
+  export let title: string | undefined = undefined;
+</script>
+
+<style>
+  .container {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    flex: 1;
+    overflow: scroll;
+  }
+
+  pre {
+    overflow: scroll;
+    font-family: var(--typeface-mono-regular);
+    font-size: 14px;
+  }
+
+  .card {
+    background-color: var(--color-foreground-level-1);
+    border-radius: 0.5rem;
+    padding: 1rem;
+    overflow: scroll;
+  }
+</style>
+
+<div class="container">
+  {#if title}
+    <h5>{title}</h5>
+  {/if}
+
+  <div class="card">
+    <pre>{JSON.stringify(data, null, 2)}</pre>
```

Comment:
Can we consolidate these into two elements?

Reply:
Not sure what you mean, could you elaborate?

Reply:
Sorry this was a bit unclear. I meant the following
```suggestion
  <div class="json">{JSON.stringify(data, null, 2)}</div>
```

Reply:
342ae9a8fde8b27774b2a9afcfc368ff835f1507